packages/editor/src/components/post-title/style.scss

```diff
@@ -24,6 +24,7 @@
 		border: $border-width solid transparent;
 		border-left-width: 0;
 		border-right-width: 0;
+		border-radius: 0;
```

Comment:
In my testing (FF, Chrome, Safari for Mac), this `border-radius` rule seems to be the only change that's needed in this file: if I remove the changes on `L10` & `L53-64`, this still seems to work just fine. Are we sure we need those updates?

Reply:
L10 is probably not needed, I'll check but the focus style is needed in the "Spotlight Mode"